src/encoded/static/scss/encoded/modules/_panels.scss

```diff
@@ -29,6 +29,27 @@ $panel-body-inset:    10px; // Number of pixels padding inside panel body
     }
 }
 
+.antibody-statuses {
+    .panel {
+        border: 0;
+    }
+}
+
+.Page, .Search, .Matrix, .ChipSeqMatrix, .Summary, .EntexMatrix, .SESCCStemCellMatrix, .ReferenceEpigenomeMatrix, .MouseDevelopmentMatrix {
+    .panel {
+        border: 0;
+    }
+}
```

Comment:
I think these pages should have a new class added to not have a border. This works, but we have to remember to add to this long selector string when we add new matrices.

Reply:
I do not get what you mean. Is this a must-have?

Reply:
Not a must have, but we might want to move to that eventually. Instead of specifying every matrix class in this selector list, and having to maintain it as we get more classes, all matrices and pages should have a class we can target and not have the border.